packages/esm-patient-labs-app/src/lab-orders/add-lab-order/useTestTypes.test.ts

```diff
@@ -0,0 +1,97 @@
+import { renderHook, waitFor } from '@testing-library/react';
+import { useTestTypes } from './useTestTypes';
+import { defineConfigSchema, getDefaultsFromConfigSchema, openmrsFetch, useConfig } from '@openmrs/esm-framework';
+import { ConfigObject, configSchema } from '../../config-schema';
```

Comment:
Looks like we can safely rid of the `defineConfigSchema` and `ConfigObject` imports.